BuildProduct.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT=$(readlink -f "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+cd $SCRIPTPATH
+
+
+VERSION="3.0.4-SNAPSHOT"
+MAVEN_QUIET=""
+INTELLIJ_VERSION=false
+
+while getopts ":hqv" option; do
+    case $option in
+        h) echo "usage: $0 [-h] [-q] [-v]"; exit ;;
+        q) MAVEN_QUIET="-q" ;;
+        v) INTELLIJ_VERSION=true ;;
+        ?) echo "error: option -$OPTARG is not implemented"; exit ;;
+    esac
+done
+
+
+ECLIPSE_KEY=${ECLIPSE_TELKEY}
+INTELLIJ_KEY=${INTELLIJ_TELKEY}
+ARTIFACTS_DIR="artifacts"
+# check dir exists
+if [ ! -d  "$ARTIFACTS_DIR" ]; then
+    echo "Creating artifacts directory $ARTIFACTS_DIR"
+    mkdir -p $ARTIFACTS_DIR
+fi
+
+# echo shell commands when they are executed.
+set -x
+
+# Build Utils
+mvn install -f $SCRIPTPATH/Utils/pom.xml -Dmaven.repo.local=$SCRIPTPATH/.repository $MAVEN_QUIET
+rm -rf /c/Signing/ToSign/*
+cp $SCRIPTPATH/Utils/azuretools-core/target/azuretools-core-${VERSION}.jar /c/Signing/ToSign/
```

Comment:
Use two variables to define ToSign and Signed directory. And reference these variables at later code.